project/Settings.scala

```diff
@@ -0,0 +1,83 @@
+import sbt._
+
+/**
+ * Application settings and dependencies
+ */
+object Settings {
+  object Definitions {
+    /** The name of the application */
+    val name = "ocs3"
+
+    /** Project version */
+    val version = "2016001.1.1"
```

Comment:
@cquiroz I had not gotten that far. [`sbt-native-packager`](https://github.com/sbt/sbt-native-packager) is worth a look ... it knows how to do startup scripts and that kind of thing.